packages/entity-example/src/entities/AllowIfUserOwnerPrivacyRule.ts

```diff
@@ -28,8 +28,10 @@ export default class AllowIfUserOwnerPrivacyRule<
   TEntity extends ReadonlyEntity<TFields, TIDField, ExampleViewerContext>,
   TSelectedFields extends keyof TFields = keyof TFields,
 > extends PrivacyPolicyRule<TFields, TIDField, ExampleViewerContext, TEntity> {
-  constructor(private readonly entityOwnerField: keyof TFields) {
+  private readonly entityOwnerField: keyof TFields;
+  constructor(entityOwnerField: keyof TFields) {
```

Comment:
Assuming `erasableSyntaxOnly` caused this? If it's just for entity-example that's fine, though we use constructor property syntax and enums most places so it should be limited to just this package.